projects/max11205pmb1/Makefile

```diff
@@ -1,6 +1,6 @@
 # Select the example you want to enable by chossing y for enabling and n for disabling
-BASIC_EXAMPLE=n
-IIO_EXAMPLE=y
+BASIC_EXAMPLE=y
```

Comment:
this changes are unrelated to the dc2903a project. please remove them, and add separate commit if needed.